modules/tinymce/src/plugins/lists/main/ts/ui/Dialog.ts

```diff
@@ -47,10 +51,17 @@ const open = (editor: Editor) => {
     ],
     onSubmit: (api) => {
       const data = api.getData();
-      editor.execCommand('mceListUpdate', false, {
-        attrs: {
-          start: data.start === '1' ? '' : data.start
-        }
+      parseStartValue(data.start).each((detail) => {
+        editor.undoManager.transact(() => {
```

Comment:
Do we need the transact if we're only doing the one thing inside of the callback?